automation/perfscale-test.sh

```diff
@@ -24,8 +24,7 @@ kubectl() { KUBEVIRTCI_VERBOSE=false cluster-up/kubectl.sh "$@"; }
 _prometheus_port_forward_pid=""
 trap "clean_up" EXIT SIGINT SIGTERM SIGQUIT
 clean_up() {
-  kill -9 $_prometheus_port_forward_pid 2> /dev/null
-  make cluster-clean
```

Comment:
What if you fail somewhere between(for example at cluster-sync)? Don't you want to clean up?

Reply:
Right, but `make cluster-clean` is failing with an unknown reason...
Having some logs can enable us to understand what is happening and include the clean again in trap later in another PR...